extension-config.json

```diff
@@ -13,6 +13,12 @@
       "path": "frontend/portals/ProductInfoAfter",
       "target": "product.info.after",
       "type": "portals"
+    },
+    {
+      "type": "portals",
+      "target": "app.globals",
+      "path": "frontend/portals/GlobalCssRules",
```

Comment:
since the file is not there anymore it should cause compiler errors. please remove this leftover from your first implementation

Reply:
Did not push the updated extension-config file because i added the yotpo key here. Removed the leftover object